dist/aframe-simple-link-component.js

```diff
@@ -90,19 +100,55 @@ AFRAME.registerComponent('simple-link', {
   /**
    * Called once when component is attached. Generally for initial setup.
    */
-  init: function () { },
+  init() {
+    this.navigate = this.navigate.bind(this);
+    const { el } = this;
+
+    el.setAttribute('geometry', {
+      primitive: 'circle',
+      radius: 1
+    });
+    if (this.data.image) {
+      el.setAttribute('material', {
+        src:
+          typeof this.data.image === 'string'
+            ? this.data.image
+            : this.data.image.src,
```

Comment:
Hmm.. I wonder if this is needed? https://aframe.io/docs/0.8.0/core/component.html#schema_asset